openpype/hosts/tvpaint/plugins/create/create_render_pass.py

```diff
@@ -29,9 +31,13 @@ def get_dynamic_data(
         dynamic_data = super(CreateRenderPass, cls).get_dynamic_data(
             variant, task_name, asset_id, project_name, host_name
         )
-        dynamic_data["render_pass"] = variant
+        dynamic_data["renderpass"] = variant
         dynamic_data["family"] = "render"
 
+        # TODO remove - Backwards compatibility for old subset name templates
+        # - added 2022/04/28
+        dynamic_data["renderpass"] = dynamic_data["render_pass"]
```

Comment:
Won't this always overwrite the `variant` value set above?

Reply:
This is called in `get_subset_name`, before creation.

Reply:
It's more that a few lines above `renderpass` is set to `variant` and then here we're directly setting it to `dynamic_data["render_pass"]`. So I have the feeling that either of the two lines is redundant.

Reply:
That's because I swapped keys :pensive: .